compiler/damlc/lib/DA/Cli/Damlc.hs

```diff
@@ -447,22 +447,24 @@ execIde telemetry (Debug debug) enableScenarioService options =
             -- performance will be significatly impacted by large log output.
             threshold
             "LanguageServer"
+          cacheDir <- getXdgDirectory XdgCache "daml"
```

Comment:
We don’t actually use the env var here do we? I would have expected this to read `DAML_CACHE` and maybe default to this or `DAML_HOME` if it’s not set.

Reply:
I think you're right, `DAML_CACHE` makes more sense.